src/main/java/de/tum/in/www1/artemis/web/rest/TextAssessmentResource.java

```diff
@@ -147,9 +153,12 @@
         }
         final var response = super.saveExampleAssessment(exampleSubmissionId, textAssessment.getFeedbacks());
         if (response.getStatusCode().is2xxSuccessful()) {
+            final var exercise = textExerciseRepository.findByIdElseThrow(exerciseId);
             final Submission submission = response.getBody().getSubmission();
             final var textSubmission = textSubmissionService.findOneWithEagerResultFeedbackAndTextBlocks(submission.getId());
-            saveTextBlocks(textAssessment.getTextBlocks(), textSubmission, textExerciseRepository.findByIdElseThrow(exerciseId), response.getBody().getFeedbacks());
+            final var feedbacksWithIds = response.getBody().getFeedbacks();
+            saveTextBlocks(textAssessment.getTextBlocks(), textSubmission, exercise, feedbacksWithIds);
+            sendFeedbackToAthena(exercise, textSubmission, feedbacksWithIds);
```

Comment:
What about changes to the feedback? For example, I give a bad review, press save, and then change/delete feedback again. How would we handle this? Athena would need the feedback removed/changed.

Reply:
I discussed this with @maximiliansoelch. Athena already gets changed feedback (it should be overridden in the Athena DB because it has the same ID). We decided that feedback deletion happens too rarely to be relevant for now, so it should be fine like this for now.